docs/Demo.md

```diff
@@ -12,37 +8,137 @@
 
 In this demo, you will get a taste of how Pelorus captures a change going through the application's delivery cycle.
 
-1. Initializing Pelorus sets the baseline by looking at existing stored data
-2. Create a new commit
-3. Watch as the metrics and trends change as new versions roll out
+* Install and configure Pelorus
+* Install a sample application that Pelorus will measure
+* Set a baseline of data for Pelorus measurements
+* Create a new commits, and Github issues
+* Watch as the metrics and trends change as new versions roll out
 
-Pelorus should be used as a conversation tool to read the trends in metrics and react by making informed investments in the software delivery process.
+Understand that Pelorus should be used as a conversation tool to read the trends in metrics and react by making informed investments in the software delivery process.
 
-## Prerequisites
+--------------------------
 
-* Pelorus is installed and running
-  * Follow the steps outlined in the [todolist install guide](../samples/todolist.md#pelorus-install-steps)
+# Lead Time for Change and Deployment Frequency
 
-* The todolist sample application is installed and running
-  * Follow the steps outlined [todolist sample install guide](../samples/todolist.md#todolist-sample-application-install-steps)
+![dora_lead_deployment](img/dora_metrics1.png)
 
-> **Note:**
-> Ensure that your github fork of mig-demo-apps is correctly set as the uri value in the todolist BuildConfig.
-```
-- kind: BuildConfig
-  apiVersion: build.openshift.io/v1
-  metadata:
-    name: todolist
-<snip>
-      source:
-        type: Git
-        git:
-          uri: https://github.com/weshayutin/mig-demo-apps.git
-          ref: master
+## Workflow - Install
+- install and configure Pelorus
+- install a sample application
 
-```
+### Pelorus install steps
+* Create the pelorus namespace
+
+        oc create namespace pelorus
+
+* Install the required granfa and prometheus operators
+
+        helm install operators charts/operators --namespace pelorus
+    
+* Install Pelorus
+    
+        helm install pelorus charts/pelorus --namespace pelorus
+
+* Check the install
+
+        oc get all -n pelorus
+    
+> **Note:** Please pause to allow the install to fully complete.
```

Comment:
don't think "pause" is valid here... maybe better to check if pods are either Running or Completed state?

```
$ oc get pods -n pelorus
NAME                                                   READY     STATUS      RESTARTS      AGE
deploytime-exporter-1-deploy                           0/1       Completed   0             93s
deploytime-exporter-1-rwk5l                            1/1       Running     0             90s
grafana-deployment-55f77ccc8f-d7m92                    2/2       Running     0             84s
grafana-operator-controller-manager-7678cc5c7c-spvls   2/2       Running     0             4m5s
prometheus-operator-559d659944-fvsjg                   1/1       Running     0             3m53s
prometheus-prometheus-pelorus-0                        3/3       Running     1 (89s ago)   93s
prometheus-prometheus-pelorus-1                        3/3       Running     1 (89s ago)   93s
```

In our e2e script we are checking for the svc services:

```
$ oc get --namespace pelorus svc 
NAME                                                  TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)             AGE
deploytime-exporter                                   ClusterIP   172.30.178.164   <none>        8080/TCP            4m16s
grafana-operator-controller-manager-metrics-service   ClusterIP   172.30.223.76    <none>        8443/TCP            6m50s
grafana-service                                       ClusterIP   172.30.249.137   <none>        3000/TCP,9091/TCP   4m15s
prometheus-operated                                   ClusterIP   None             <none>        9090/TCP            4m15s
prometheus-pelorus                                    ClusterIP   172.30.90.84     <none>        9090/TCP,9091/TCP   4m16s
```